src/components/20-molecules/datepicker/react/DemoDatepickerOnInputfieldKeyUp.jsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import AXADatepickerReact from './AXADatepickerReact';
+
+const DemoDatepickerOnInputfieldKeyUp = () => {
+  return (
+    <div>
+      <span id="datepicker-react-testoutput" />
+      <br />
+      <AXADatepickerReact
+        inputfield
+        id="datepicker-react-inputfield-on-date-change"
+        onInputfieldKeyUp={e => {
+          document.querySelector('#datepicker-react-testoutput').innerHTML =
+            e.key;
```

Comment:
it's a little confusing just to see a letter above the input field without additional text. I would add some text like: `last typed letter: xy`